packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -234,28 +234,30 @@ class MinimumTextContrastGuideline extends AccessibilityGuideline {
       // contrast ratio based on text size/weight.
       double? fontSize;
       bool isBold;
+      // Italic ahem font has AA artifacts between characters and requires an expanded rect.
+      bool forceLargeRect = false;
```

Comment:
I did find one issue, is that we hit AA artifacts on ahem if it is itallic. We should investigate a way to disable AA globally for these sorts of tests